docs/products/kafka/howto/public-access-in-vpc.rst

```diff
@@ -0,0 +1,14 @@
+Enable public access in a VPC
+==============================
+
+To enable public access for **Aiven for Apache Kafka** service which is running within a virtual private cloud (VPC), follow these steps:
```

Comment:
Public Access is not only for Kafka, could this be a generic example (or specify this example uses Kafka but we support it in other services, as well as service components)

Reply:
Great point. Refactored to make generic